tests/generic_relations/tests.py

```diff
@@ -490,6 +490,15 @@ def test_subclasses_with_gen_rel(self):
         TaggedItem.objects.create(content_object=granite, tag="countertop")
         self.assertEqual(Rock.objects.filter(tags__tag="countertop").count(), 1)
 
+    def test_subclasses_with_parent_gen_rel(self):
+        """
+        Generic relations on a base class (Vegetable) work correctly in
+        subclasses (Carrot).
+        """
+        bear = Carrot.objects.create(name='carrot')
+        TaggedItem.objects.create(content_object=bear, tag='orange')
+        self.assertEqual(Carrot.objects.filter(tags__tag='orange').count(), 1)
```

Comment:
I would either use `self.assertTrue(Carrot.objects.filter(tags__tag='orange').exists())` or `self.assertEqual(Carrot.objects.get(tags__tag='orange'), bear)` but otherwise LGTM.